lib/cogctl/actions/chat_handles/create.ex

```diff
@@ -3,9 +3,9 @@ defmodule Cogctl.Actions.ChatHandles.Create do
   alias Cogctl.Table
 
   def option_spec do
-    [{:user, ?u, 'user', :string, 'Username of user to add handle to', :required},
```

Comment:
That was a previous implementation @christophermaier. We were specifying explicitly which options were required. That was before I realized that `getopt` has it's own idea of required options, any option without a default value.